testsuite/makefiles/Makefile.several

```diff
@@ -108,9 +108,15 @@ run-file:
 	fi \
 	&& \
 	if [ -f $$F.checker ]; then \
-	  DIFF="$(DIFF)" SORT="$(SORT)" sh $$F.checker; \
+	  DIFF="$(DIFF)" SORT="$(SORT)" sh $$F.checker || { \
+	    printf "  Error: output checker failed!\n"; \
+	    exit 1; \
+	  }; \
 	else \
-	  $(DIFF) $$F.reference $$F.result >/dev/null; \
+	  $(DIFF) $$F.reference $$F.result >/dev/null || { \
+	    printf "  Error: results don't match reference output!\n"; \
+	    exit 1; \
```

Comment:
It corresponds to that command (lines 110 - 118) to return an error code of 1, same as the diff did originally. It just preserves the error so that make will still abort the target like it did before. Other tests still run.